feature/account/common/src/main/kotlin/app/k9mail/feature/account/common/ui/AccountTopAppBar.kt

```diff
@@ -6,11 +6,16 @@ import androidx.compose.ui.Modifier
 import androidx.compose.ui.res.stringResource
 import app.k9mail.core.ui.compose.common.DevicePreviews
 import app.k9mail.core.ui.compose.designsystem.organism.TopAppBar
-import app.k9mail.core.ui.compose.theme.K9Theme
 import app.k9mail.core.ui.compose.theme.MainTheme
-import app.k9mail.core.ui.compose.theme.ThunderbirdTheme
+import app.k9mail.core.ui.compose.theme.PreviewWithThemes
 import app.k9mail.feature.account.common.R
 
+/**
+ * Top app bar for the account screens.
+ *
+ * @param title The title of the screen.
+ * @param modifier Modifier for styling.
```

Comment:
The documentation for the parameters doesn't add much value compared to just the parameter names. In cases like this I think it's fine to not document the parameters.